recipes/glib/find-libintl.h.patch

```diff
@@ -0,0 +1,8 @@
+--- glib-2.0.pc.in.orig	2015-12-23 14:06:46.000000000 -0800
++++ glib-2.0.pc.in	2015-12-23 14:07:07.000000000 -0800
+@@ -13,4 +13,4 @@
+ Requires.private: @PCRE_REQUIRES@
+ Libs: -L${libdir} -lglib-2.0 @INTLLIBS@
+ Libs.private: @G_THREAD_LIBS@ @G_LIBS_EXTRA@ @PCRE_LIBS@ @INTLLIBS@ @ICONV_LIBS@ @CARBON_LIBS@ @COCOA_LIBS@
+-Cflags: -I${includedir}/glib-2.0 -I${libdir}/glib-2.0/include @GLIB_EXTRA_CFLAGS@
++Cflags: -I${includedir}/glib-2.0 -I${libdir}/glib-2.0/include -I${includedir} @GLIB_EXTRA_CFLAGS@
```

Comment:
Do we still need this patch? It seems like we are just grabbing some CFLAGS. Does an environment work just as well here?